lib/src/zap/zap.c

```diff
@@ -136,6 +136,7 @@ static char *__zap_event_str[] = {
 	"ZAP_EVENT_READ_COMPLETE",
 	"ZAP_EVENT_WRITE_COMPLETE",
 	"ZAP_EVENT_RENDEZVOUS",
+	"ZAP_EVENT_SEND_COMPLETE",
```

Comment:
This is the time to add the send completion. We would have ZAP_EVENT_SEND_COMPLETE, and ZAP_EVENT_SEND_MAPPED_COMPLETE.

Reply:
Got it.